### PR TITLE
addresses https://github.com/ubtue/DatenProbleme/issues/2119

### PR DESCRIPTION
**Pull request by mara-spiess**

### CODE REVIEW

**Comment by jriedl** [comment]
Please make explicit that it is now based on the current version of the main translators repository.

**Comment by mara-spiess** [comment]
Would amending the commit message be sufficient or should I add a comment in the code?

**Comment by jriedl** [comment]
Commit message is sufficient.

File: ubtue_Open Journal Systems Standard.js [hunk 1]
```diff
@@ -337,7 +337,7 @@ function invokeEMTranslator(doc) {
 			}
 		}
 		if (["1988-4265"].includes(i.ISSN)) {
-			if (i.title.match(/\d{4},\s+\d+\s+págs.,?\s+isbn:\s+[\d\-x]+/i)) {
+			if (i.title.match(/\s+isbn:?\s+[\d\-x]+/i)) {
```
Comment (jriedl) [inline ubtue_Open Journal Systems Standard.js:340]: The now removed part with `págs` was only added yesterday in commit `b9a8d24`. Is the removal intentional?

**Pull request merged**